src/appshell/view/preferences/preferencesmodel.cpp

```diff
@@ -131,17 +131,13 @@ QString PreferencesModel::currentPageId() const
     return m_currentPageId;
 }
 
-void PreferencesModel::load(const QString& currentPageId)
+void PreferencesModel::load()
```

Comment:
I don't quite understand how loading a custom page will work. Perhaps this logic is now in qml? It would be better if it were here again